modules/plugins/languages/kotlin.nix

```diff
@@ -0,0 +1,75 @@
+{
+  config,
+  pkgs,
+  lib,
+  ...
+}: let
+  inherit (lib.options) mkEnableOption mkOption;
+  inherit (lib.modules) mkIf mkMerge;
+  inherit (lib.types) package;
+  inherit (lib.nvim.types) mkGrammarOption;
+  inherit (lib.lists) isList;
+  inherit (lib.nvim.lua) expToLua;
+
+  cfg = config.vim.languages.kotlin;
+
+  # Creating a version of the LSP with access to the kotlin binary.
+  # This is necessary for the LSP to load the standard library
+  kotlinLspWithRuntime = pkgs.symlinkJoin {
+    name = "kotlin-language-server-with-runtime";
+    paths = [
+      pkgs.kotlin-language-server
+      pkgs.kotlin
```

Comment:
Is kotlin-language-server not wrapped with kotlin?

Reply:
Not that I'm aware of